raiden/network/transport/udp/udp_transport.py

```diff
@@ -208,7 +207,7 @@ def __init__(self, discovery, udpsocket, throttle_policy, config):
     def start(
             self,
             raiden: RaidenService,
-            queueids_to_queues: typing.Dict[QueueIdentifier, typing.List[Message]],
+            queueids_to_queues: typing.QueueIdsToMessages,
```

Comment:
The typing change is a bit confusing with the current variable naming.

You have `queueids_to_queues` being of type `QueueIdsToMessages` while [here](https://github.com/raiden-network/raiden/pull/2458/files#diff-a6042df3a2b6bca66372154b6cceff89R266) you have the same named variable being `QueueIdsToEvents`

Can you also edit the names to align with the types?

Reply:
what are the names you want?

Reply:
Something close to the type. e.g. `queueis_to_messages` above. To avoid confusion if the type annotation is not seen.